components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -677,6 +677,9 @@
 .error_updating_consents {
   margin-top: 1em;
   margin-left: 2em;
+}
+.view-title-required-not-granted {
+  font-size: 1.15em;
```

Comment:
I would not display the top warning in a view mode at all. It is a duplication of information that not all consents are granted.